### DISCUSSION TITLE
Issue with position of inbox

### DISCUSSION THREAD

**Discussion by RuslanHryn** [magicbell/community#223] [General]
The issue with the position of the inbox has appeared after upgrading `magicbell-react` from 10.11 to 11.1.
Position with default settings:
![image](https://github.com/user-attachments/assets/76f57fd2-b029-4aad-8eb9-26846405dbd1)

With reduced window size
![image](https://github.com/user-attachments/assets/990a2c2a-ea09-4872-a99c-ba8931eaa564)

styles:

![Screenshot 2024-12-11 at 19 30 39](https://github.com/user-attachments/assets/7ae53f4d-7c74-4335-b446-23b357ae7fb2)

When setting any `placement` in  `FloatingNotificationInbox`

![image](https://github.com/user-attachments/assets/499e39b5-9647-4fcf-a812-d6345101249c)

as you can see it fixed the position of the inbox but the arrow is still in the incorrect position 

Here are styles:
![image](https://github.com/user-attachments/assets/9f9c0b63-d169-410d-8325-75581a7851d3)

The issue is reproducible on the latest Chrome and Safari

**Comment by smeijer** [comment]
Hi Ruslan,

I can't debug your positioning issues using the screenshots you provided, but please check the `placement` property, and try making one of the parents `relative` if the `absolute` positioned floating inbox seems to be positioned incorrectly.

I've pushed an update that fixes the arrow positioning. It should be better now. I've also added the `offset` and `arrowPadding` props to the `FloatingNotificationInbox` component, which makes it easier to adjust the inbox's position relative to its trigger. I hope that helps. You can use those props like:

```
<FloatingNotificationInbox 
  height={450} 
  {...props} 
  placement="bottom-start" 
  offset={24} 
  arrowPadding={16} 
  isOpen
 />
```

**Reply by RuslanHryn** [reply]
@smeijer thanks a lot! It fixed this issue
